contracts/libs/Actions.sol

```diff
@@ -23,65 +23,65 @@ library Actions {
     }
 
     struct ActionArgs {
-        /// @notice The type of action that is being performed on the system
+        // The type of action that is being performed on the system
         ActionType actionType;
-        /// @notice The address of the account owner
+        //  The address of the account owner
         address owner;
-        /// @notice The address which we move assets from or to (depending on the action type)
+        // The address which we move assets from or to (depending on the action type)
         address sender;
-        /// @notice The asset that is to be transfered
+        // The asset that is to be transfered
         address asset;
-        /// @notice The index of the vault that is to be modified (if any)
+        // The index of the vault that is to be modified (if any)
         uint256 vaultId;
-        /// @notice The amount of asset that is to be transfered
+        // The amount of asset that is to be transfered
         uint256 amount;
-        /**
-         * @notice Each vault can hold multiple short / long / collateral assets. In this version, we are restricting the scope to only 1 of each.
+        /*
+         * Each vault can hold multiple short / long / collateral assets. In this version, we are restricting the scope to only 1 of each.
          * In future versions this would be the index of the short / long / collateral asset that needs to be modified.
          */
         uint256 index;
-        /// @notice Any other data that needs to be passed in for arbitrary function calls
+        // Any other data that needs to be passed in for arbitrary function calls
         bytes data;
     }
 
     struct OpenVaultArgs {
-        /// @notice The address of the account owner
+        // The address of the account owner
         address owner;
     }
 
     struct DepositArgs {
-        /// @notice The address of the account owner
+        // The address of the account owner
         address owner;
-        /// @notice The index of the vault to which the asset will be added
```

Comment:
@antoncoding should these be @dev?

Reply:
Compiler will complain when we use either @notice or @dev in struct. Those are meant for state variables.